routes/todo-route.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const router = express.Router();
+
+const {createTodo, getAllTodos,getTodoByUserId,getTodoByTodoId, deleteTodo,deleteTodoByUserId, updateTodo} = require('../controller/todo-controller');
```

Comment:
Please export all functions in object and then import that object in this route file.